libdevcore/FixedHash.h

```diff
@@ -91,35 +85,15 @@ class FixedHash
 
 	// The obvious comparison operators.
 	bool operator==(FixedHash const& _c) const { return m_data == _c.m_data; }
-	bool operator!=(FixedHash const& _c) const { return m_data != _c.m_data; }
 	bool operator<(FixedHash const& _c) const { for (unsigned i = 0; i < N; ++i) if (m_data[i] < _c.m_data[i]) return true; else if (m_data[i] > _c.m_data[i]) return false; return false; }
```

Comment:
If you use `FixedHash` in any sorted STL container, for example `std::map`, you need `operator<` (and only that).